pinot-plugins/pinot-stream-ingestion/pinot-pulsar/src/main/java/org/apache/pinot/plugin/stream/pulsar/PulsarMessageBatch.java

```diff
@@ -49,11 +49,16 @@ public int getMessageCount() {
     return _messageList.size();
   }
 
+  @Deprecated
   @Override
   public PulsarStreamMessage getMessageAtIndex(int index) {
     return _messageList.get(index);
   }
 
+  @Override
+  public byte[] getMessageBytesAtIndex(int index) {
+    return getMessageAtIndex(index).getValue();
```

Comment:
(minor) Let's not use the deprecated method
```suggestion
    return _messageList.get(index).getValue();
```